Script/AtomicEditor/editor/Preferences.ts

```diff
@@ -77,6 +86,7 @@ class Preferences {
     }
 
     write(): void {
+        if (this._configCorrupted) return;
```

Comment:
We should save, otherwise you'll get into a situation where prefs will never be saved unless you hand edit them.  If you do hand edit them and break the JSON, that is your problem :)  I do not believe we need the _configCorrupted member, and the log should say "Editor preference file invalid, regenerating default configuration"